test/EFCore.MySql.FunctionalTests/Northwind.sql

```diff
@@ -270,6 +270,8 @@ CREATE TABLE `Products` (
 /* GO */
  CREATE  INDEX `SuppliersProducts` ON `Products`(`SupplierID`);
 /* GO */
+ CREATE FULLTEXT INDEX IF NOT EXISTS `FTI__Name` ON `Products`(`Name`)
+/* GO */
```

Comment:
It's probably not a good idea to extend the standard Northwind script. Instead, just create your own simple context (or copy an existing one) like many samples do.

Reply:
I removed changes from Northwind.sql, but not really sure where to create my own context, can you give me an example or point in right direction?

Reply:
There are basically two approaches:

You can create your own `SharedStoreFixtureBase<PoolableDbContext>` derived fixture class, where you create your own data model. This gives you full control.
A random example would be the `OwnedQueryMySqlTest` class, that uses the `OwnedQueryMySqlFixture` class, that is derived from `RelationalOwnedQueryFixture`, which is derived from `OwnedQueryFixtureBase` and defines the data model (and finally derives from `SharedStoreFixtureBase<PoolableDbContext>`).
This can be done however, with just one fixture class directly deriving from `SharedStoreFixtureBase<PoolableDbContext>`.

The other way is to do something similar to `EscapesMySqlTest`, `EscapesMySqlFixture` and `EscapesMySqlFixtureBase`.
With `EscapesMySqlFixtureBase`, we just reuse some already defined `DbContext` (in this case `MusicStoreContext`). We then tailor the context to our needs (you can override `OnModelCreating()` to change the model) and change the `StoreName` property (so that a new database is created with our changed model and we don't change the behavior of existing tests in other classes).